PlottingScripts/Basic_Plot.py

```diff
@@ -0,0 +1,128 @@
+"""This is a basic script for plotting an analysis file."""
+from collections import OrderedDict
+
+import numpy as np
+import pandas as pd
+import pdb
+
+from bokeh.plotting import figure, show, output_file
+
```

Comment:
I am adding the changes that rely on your files as comments because I need to merge with your branch first before it will take effect. But it should hopefully be a quick change when we merge branches